pom.xml

```diff
@@ -36,8 +18,12 @@
     </properties>
     <dependencies>
         <dependency>
-            <groupId>net.smartcosmos</groupId>
```

Comment:
We can research the framework incompatibility later - now it is just important to have this service functional again to get other stuff done.
This user details service will never run in production environments, so IMO the framework issue does not have top priority here.